react-native-community/react-native-maps
# TestID's for e2e automation

It seems that the following pull https://github.com/react-community/react-native-maps/pull/1170 request has been abandoned, but testID is still required if one wants to make e2e tests



### Does any other open PR do the same thing?
https://github.com/react-community/react-native-maps/pull/1170 

 - the previous PR is stalled, as it's really old and the author didn't continue working on it
 - there are conflicts with the `master` branch and the author didn't fix them

### What issue is this PR fixing?
rendered markers don't have testID prop which is necessary for e2e tests
Closes https://github.com/react-community/react-native-maps/issues/2252
Closes https://github.com/react-community/react-native-maps/issues/1104
### How did you test this PR?
1. Create a marker and add it a testID
2. Launch your app in ios simulator, then in XCode navigate to the debug navigator and press the debug View Hierarchy button. This will create a snapshot of your UI similar to this:
<img width="639" alt="grafik" src="https://user-images.githubusercontent.com/693770/39981815-73f723ec-5752-11e8-8877-5a8ab9d4dea2.png">


Select the marker and it should have an identifier set in the Accessibility section on the right.

This was tested on a real device and simulator. The change applies to ios only.
